must/must.go

```diff
@@ -0,0 +1,15 @@
+package must
+
+// Do panics if err is not nil, otherwise returns t.
+// It is useful in wrapping a two-value function call
+// where you know statically that the call will succeed.
+//
+// Example:
+//
+// url := must.Do(url.Parse("http://example.com"))
```

Comment:
should this be an actual `godoc` example?
https://go.dev/blog/examples

Reply:
Good idea. I tried this just now and ran into some problems. The `ExampleDo` function isn't detected as a test, which I tracked down to an issue with generic parameters; if I removed the generic parameter I was able to make it work.

I think overall it's probably not worth tracking down a fix; though it is a good idea to add a test!